css/offcanvas.css

```diff
@@ -57,3 +57,26 @@ footer {
     width: 50%; /* 6 columns */
   }
 }
+@media screen and (max-width: 995px){
```

Comment:
You're on the right track! The album tiles are working for all screen sizes. But The singles' list is not visible for ipad or mobile devices. To check, you can use your browser developer tools or resize your screen.

Reply:
should i make singles list below the album tiles for small screen sizes?

Reply:
Yes, that would be nice. You can also try to make it horizontal, but that's optional for now.